userstats.py

```diff
@@ -90,6 +90,8 @@ def unicode_name(self):
             return self.user_name
 
 if __name__ == '__main__':
+    from dbconfig import engine
```

Comment:
Why not use get_engine instead? That ensures the engine is re-created each time, to avoid issues like this one:
https://phabricator.wikimedia.org/T178843

Reply:
Ok, I will update the pull request accordingly.

Reply:
In this case I would also suggest to make the variable `engine` private (as private as python can go) to the `dbconfig` module.

Reply:
Done!